js/modelData.js

```diff
@@ -0,0 +1,91 @@
+const projectData = [
+  {
+    snapshot: 'req/tonic.png',
+    projectname: 'Tonic',
+    technologies: ['HTML', 'CSS', 'JAVASCRIPT', 'Github', 'Ruby', 'Bootstrap'],
+    dot: 'req/dot.png',
+    para: 'A daily selection Of privately personalized reads; no accounts or sign-ups',
+    frame: ['CANOPY', 'Back End Dev', '2015'],
+    smBtnText: 'See Project',
+    popimage: 'req/poppic.svg',
+    disdes: `Lorem Ipsum is simply dummy text of the printing and typesetting
+        industry. Lorem Ipsum has been the industrys standard dummy text
+        ever since the 1500s, when an unknown printer took a galley of type
+        and scrambled it 1960s with the releaLorem Ipsum is simply dummy
+        text of the printing and typesetting industry. Lorem Ipsum has been
+        the industrys standard dummy text ever since the 1500s, when an
+        unknown printer took`,
+    sep: 'req/sep.png',
+    livelogo: 'req/live-icon.svg',
+    sourcelogo: 'req/github-blue.svg',
+    livelink: 'https://muneeb3123.github.io/project-protfolio.github.io/ ',
+    sourcelink: 'https://github.com/muneeb3123/project-protfolio.github.io',
+  },
```

Comment:
Great job using javascript objects and modular code! Kudos for this 🚀

Reply:
thanks